Traceback (most recent call last):
  File "/home/liuchi/yitianjiao/aamas2025/free-range-zoo/free_range_zoo/experiments/train.py", line 46, in <module>
    from free_range_zoo.envs.wildfire.baselines import GenerateAgent,StrongestBaseline
  File "/home/liuchi/yitianjiao/aamas2025/free-range-zoo/free_range_zoo/envs/wildfire/baselines/__init__.py", line 5, in <module>
    from free_range_zoo.envs.wildfire.baselines.generated_agent import GenerateAgent
  File "/home/liuchi/yitianjiao/aamas2025/free-range-zoo/free_range_zoo/envs/wildfire/baselines/generated_agent.py", line 145, in <module>
    python
NameError: name 'python' is not defined
